core/templates/dev/head/pages/exploration_player/learner_view_suggestion_modal_directive.html

```diff
@@ -3,15 +3,17 @@ <h3>Suggest a Change</h3>
 </div>
 
 <div class="modal-body protractor-test-exploration-suggestion-modal">
-  <text-angular-rte ng-show="showEditor" html-content="suggestionHtml"></text-angular-rte>
+  <!-- If there is a ng-model, you have to have a dot in there. -->
+  <ck-editor-rte ng-if="isCurrentRTECKEditor" ng-show="showEditor" ng-model="suggest.suggestionHtml"></ck-editor-rte>
+  <text-angular-rte ng-if="!isCurrentRTECKEditor" ng-show="showEditor" html-content="suggestionHtml"></text-angular-rte>
```

Comment:
Why have both suggest.suggestionHtml and suggestionHtml? Shouldn't these two RTEs refer to the same thing?

Also, I think you should get rid of the isCurrentRTECKEditor flag and just remove the textAngular stuff. Once this PR is in, there's no going back, so there's no point having the switch.